logger/logger.go

```diff
@@ -88,6 +88,7 @@ type Logger struct {
 	stdEnabled     bool
 	authEnabled    bool
 	reqEnabled     bool
+	excludePath    string
```

Comment:
I was kind of expecting this to be a list of paths to exclude, WDYT?

Reply:
Mm. Will do.

Reply:
Done now.